examples/prometheus/README.md

```diff
@@ -0,0 +1,135 @@
+# Monitoring STUNner with Prometheus and Grafana
+
+This tutorial demonstrates monitoring of a STUNner instance with Prometheus and Grafana.
+
+In this demo you will learn how to:
+* monitor STUNner with Prometheus
+* how to visualize STUNner metrics using Grafana
+
+## Installation
+
+### Prerequisites
+
+* The tutorial assumes a fresh STUNner installation in a namespace called `stunner`; see the [STUNner installation and configuration guide](/doc/INSTALL.md).
+* You need a web browser to run this tutorial.
+
+### Setup
+
+After installing STUNner, enable the STUNner metrics endpoint:
+```console
+kubectl -n stunner patch gatewayconfigs.stunner.l7mp.io stunner-gatewayconfig --patch '{"spec": {"metricsEndpoint": "http://0.0.0.0:8080/metrics" }}' --type=merge
+```
+
+Patch STUNner deployment to enable the monitoring port (required by the pod monitor):
```

Comment:
IMO it would be better to modify the stunner helm chart itself and add an extra value (`.values.stunner.deployment.monitoring.enabled=false`) to [Values.yaml](https://github.com/l7mp/stunner-helm/blob/main/helm/stunner/values.yaml). This way users could enable monitoring while installing. 
The highlighted step could come in handy for already installed charts. OR use [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/). 
I don't know whether patching or helm upgrading the deployment will restart the corresponding pods, but restarting STUNner pods with running calls could cause some trouble.